class_person.rb

```diff
@@ -18,6 +21,12 @@ def can_use_services?
     of_age? || @parent_permission
   end
 
+  def add_rental(book, date)
+    rental = Rental.new(date, book, self)
+    @rentals.push(rental)
+    rental
```

Comment:
- [ ] Please note that when you create a new rental, it automatically adds itself to the person's rentals thanks to this code `person.rentals << self` in the `Rental` class at line `9`. That means we don't need to push the new rental to @rentals after it has been created. Therefore I suggest you remove this code from the `add_rental` method. 
  Example:
```ruby
def add_rental(book, date)
  Rental.new(date, book, self)
end
```

Reply:
- Remove unnecessary code from `add-rental` method to resolve bug.